app/main.py

```diff
@@ -1,13 +1,41 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
```

Comment:
it's preferable to use get() method instead of parenthesis everywhere for getting value by key to avoid KeyError

Reply:
why mentor didn't use get() method in this case? 
![image](https://github.com/mate-academy/py-linter-formatter/assets/130069561/3b4bc03a-4126-4da9-a430-46a966209950)